testing/rabbitmq/rabbitmq.initd

```diff
@@ -0,0 +1,38 @@
+#!/sbin/openrc-run
+
+pidfile=/var/run/rabbitmq/rabbitmq.pid
+name=rabbitmq
+daemon=/usr/sbin/rabbitmq-server
+
+depend() {
+	need net localmount
+	after firewall
+}
+
+start() {
+	ebegin "Starting ${name}"
+	# We need to test if /var/run/rabbitmq exists, since scirpt will fail if it does not
+	[ ! -e /var/run/rabbitmq ] && \
+		mkdir /var/run/rabbitmq && \
+		chown rabbitmq:rabbitmq /var/run/rabbitmq
+
+    
+    
+	start-stop-daemon --start \
+		--env RABBITMQ_PID_FILE=$pidfile \
+		--pidfile $pidfile \
+		--exec /bin/sh -- -c "$daemon \
+		> /var/log/rabbitmq/startup_log \
+		2> /var/log/rabbitmq/startup_err 0<&- &"
```

Comment:
This is not systemd, but OpenRC that is very different. Using `&` here really doesn't make any sense and it's not acceptable.

However, as I looked at RabbitMQ, it'll be a bit more complicated. I'll rewrite this init script myself, so don't make a worry about it.